content/concepts/interoperability/cross-chain-bridges.md

```diff
@@ -0,0 +1,128 @@
+* Locking chain: The chain on which the bridge locks and unlocks assets.
+
+* Issuing chain: The chain on which the bridge mints and burns assets.
+
+* Door account: A special type of account that is used to move assets from one chain to another. The door account on a locking chain is used to put assets into trust, and the door account on an issuing chain used to issue wrapped assets. 
+
+* Cross-chain claim ID: A special identifier used for cross-chain transfers. A cross-chain claim ID represents *one* cross-chain transfer of value.
+
+## How Do Cross-Chain Transactions Work?
+
+### Pre-requisites
+
+You must ensure that the following are set up and running before initiating cross-chain transactions. 
+
+* Ensure that the witness server(s) are up and running.
+* Set up a bridge between the two chains, including a _door account_ on each chain. On one chain, the asset is locked and unlocked, hence the name "locking chain", and on the other chain, assets are minted and burned, or issued and reclaimed, hence the name "issuing chain".
+* When using the XRP Ledger Mainnet as one of the chains, ensure that  the `Sidechains` amendment is enabled in the `rippled.cfg` configuration file. 
+
+### Working of Cross-Chain Transactions 
+ 
+At a high-level, cross-chain transactions involve the following steps: 
+
+1. Claim a cross-chain claim ID on the issuing chain.
+2. Submit a cross-chain transfer transaction on the locking chain, attaching the claimed cross-chain claim ID and include a reward for the witness servers. This locks the asset on the locking chain.
+3. Obtain the attestations from the witness servers that the transaction occurred on the issuing chain.
+4. Submit a transaction claim for the transferred value on the issuing chain, attaching the attestation as proof that the value was indeed transferred.
+5. The rewards are then distributed to the witness servers' accounts on the issuing chain.
+
+Consider an example where Alice wants to send XRP from her account on the XRP Ledger Mainnet (locking chain) to her account sAlice on a sidechain (issuing chain).
+
+* A bridge that transfers XRP between the XRP Ledger Mainnet and the sidechain has been set up with XChainCreateBridge.
+* Witness servers are up and running.
+* Alice has also already run a XChainCreateAccountCommit transaction at some point, so she has an account on the sidechain.
+
+<!-- Add image of just the bridge created-->
+
+![Cross-chain Transactions](img/xrpl-bridging-solution.png "Cross-chain transactions")
+
+1. sAlice first checks out a claim ID with XChainCreateClaimID on the sidechain, specifying the above bridge. She retrieves the claim ID from the transaction metadata or the `xchaincreateclaimid` RPC call.
+
+2. Alice then takes the cross-chain claim ID from sAlice’s XChainCreateClaimID transaction and submits a XChainCommit transaction on the Mainnet with that claim ID, locking up a specified amount of XRP. She specifies sAlice’s account in the OtherChainDestination field.
```

Comment:
This step is the first of a few cases that refer to "Mainnet" (which traditionally means the one production XRP Ledger) whereas the previous explanations described things in terms of an "issuing chain" and a "locking chain". I believe that the thought is that Mainnet would most often be the locking chain for production sidechains, but other chains could be the locking chain in some configurations especially in test cases.

This should probably be consistent with the use of "locking chain" / "issuing chain" rather than Mainnet / sidechain, I think.

Reply:
Agreed - I think it's fine to say "mainnet is the locking chain, sidechain is the issuing chain" at the top, and then just stick to locking chain/issuing chain terminology